guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -22,9 +22,7 @@ Configure an LDAP authentication source to enable users to log in to {Project} w
 ----
 . In the {ProjectWebUI}, navigate to *Administer* > *Authentication Sources*.
 . From the *LDAP* menu, select *Create*.
-. On the *LDAP server* tab, enter the details of your LDAP server.
-+
-For TLS encrypted connections, select *LDAPS* to enable encryption.
+. On the *LDAP server* tab, enter the details of your LDAP server. For TLS encrypted connections, select *LDAPS* to enable encryption.
```

Comment:
I think the rule is missing this occurrence because the second sentence does not end with a punctuation mark + space + character. It ends with a line break.